From 9228d7c87b6affc60fa784ca44ac26710b28abd9 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Wed, 8 Sep 2021 09:31:15 +0300
Subject: [PATCH] tests: disable failing tests

Some of the tests fail to link with gold linker used.

Upstream-Status: Inappropriate [embedded specific]
---
 tests/auto/gui/image/CMakeLists.txt  | 2 +-
 tests/auto/gui/math3d/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/auto/gui/image/CMakeLists.txt b/tests/auto/gui/image/CMakeLists.txt
index 27916d0409..cf6c86e8dc 100644
--- a/tests/auto/gui/image/CMakeLists.txt
+++ b/tests/auto/gui/image/CMakeLists.txt
@@ -8,7 +8,7 @@ if(QT_FEATURE_ico)
     add_subdirectory(qicoimageformat)
 endif()
 add_subdirectory(qpixmap)
-add_subdirectory(qimage)
+#add_subdirectory(qimage)
 add_subdirectory(qimageiohandler)
 add_subdirectory(qimagewriter)
 add_subdirectory(qmovie)
diff --git a/tests/auto/gui/math3d/CMakeLists.txt b/tests/auto/gui/math3d/CMakeLists.txt
index 11c50163ad..bc3ce532c8 100644
--- a/tests/auto/gui/math3d/CMakeLists.txt
+++ b/tests/auto/gui/math3d/CMakeLists.txt
@@ -1,5 +1,5 @@
 # Generated from math3d.pro.
 
-add_subdirectory(qquaternion)
+#add_subdirectory(qquaternion)
 add_subdirectory(qvectornd)
 add_subdirectory(qmatrixnxn)
